ndarbas14/iveskAutomobili.php

```diff
@@ -0,0 +1,128 @@
+<?php
+session_start();
+require_once 'Radar.php';
+require_once 'Filtras.php';
+require_once 'startRadarSes.php';
+
+if (!isset($_SESSION['automobiliai'])) {
+    $_SESSION['automobiliai'] = [];
+
+}
+
+if (!isset($filtr)) {
+    if (isset($_POST['filtras'])) {
```

Comment:
Filtrams dazniausiai yra naudojamas get metodas.

Reply:
Ar filtrams saugumo reikalavimai  negalioja?

Reply:
Saugumo reikalavimai galioja viskam. GET metodas yra naudojamas dazniausiai todel, nes norima filtro reiksme persinesti einant i kita puslapi ir pan. Tarkim issifiltravote automobilius kur numeris prasideda A raide ir ju yra nerealiai daug. Jus turbut padarysite puslapiavima. Einant i antra puslapi jus prarasite duomenis jei filtras bus POST metodu tik tikrinimas. Taip pat, gal isfiltravote automobilius ir norite nusiusti draugui ka isfiltravote, tai tiesiog nukopijuojate nuoroda ir viskas. Jei yra POST metodas - to padaryti nepavyks.